turbinia/client.py

```diff
@@ -387,21 +387,29 @@ def get_task_data(
             '{2!s}'.format(retry_count, MAX_RETRIES, exception))
         retry_count += 1
 
-      if response is None:
+      if not response:
+        retry_count += 1
+        time.sleep(RETRY_SLEEP)
+      elif response.get('error') and response.get('error').get('code') == 503:
```

Comment:
Can this be just be `elif response.get('error').get('code') == 503`?

Reply:
This is actually slightly safer as is because if `response['error]` doesn't exist, then the second expression in the line would cause an `AttributeError` when evaluated (because `None` doesn't have a `.get()` method).   I could actually change it to something quite similar to that though:  `elif response.get('error', {}).get('code') == 503:`.  I'll make the change.

Reply:
Ah yeah was also thinking it was due to order of operations but mentioned just to be sure. Yeah SGTM